source/MaterialXFormat/Util.cpp

```diff
@@ -137,4 +137,73 @@ StringSet loadLibraries(const FilePathVec& libraryFolders,
     return loadedLibraries;
 }
 
+void resolveFileNames(DocumentPtr doc, const FileSearchPath& searchPath, StringResolverPtr customResolver)
+{
+    // Set value to name + file prefix.
+    for (ElementPtr elem : doc->traverseTree())
+    {
+        ValueElementPtr valueElem = elem->asA<ValueElement>();
+        if (!valueElem)
+        {
+            continue;
+        }
+        if (valueElem->getType() != FILENAME_TYPE_STRING)
```

Comment:
This "if" can be merge with the one above.

Reply:
Will do.